exercises/practice/bob/.articles/performance/content.md

```diff
@@ -7,19 +7,21 @@ The [approaches page][approaches] lists two idiomatic approaches to this exercis
 1. [Using `if` statements][approach-if].
 2. [Using `match` on a `tuple`][approach-match].
 
-For our performance investigation, we'll also include a third approach that [uses an answer array][approach-answer-array].
+For our performance investigation, we'll also include a third approach that [uses an answer array][approach-answer-array] and
+a fourth that uses a state machine.
```

Comment:
```suggestion
For our performance investigation, we'll also include a third approach that [uses an answer array][approach-answer-array] and a fourth that uses a state machine.
```

Our convention in markdown is [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line).